charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,94 @@
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project.  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports.  
+__Onboarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP.  
+__Scoping:__ The process of investigating the available resources and knowledge in a community around a specific assay to better understand how much work it would take to make a pipeline to process data from that assay.  
+__Standardized Pipeline:__ An official pipeline of the HCA project.  
+__Transformative Analysis:__ The processing of data that changes data or creates new data. This would not include processes that solely subset or reorganize data.
+
+## In-scope
+
+### Pipeline Creation
+
+* Scoping or review of scoping for new pipelines  
+* Suggesting, but not selection of, new standardized pipelines for the HCA DCP  
+* Creating all standardized HCA pipelines and documenting them  
+* Reviewing, on-boarding, and potentially creating community pipelines  
+* Reviewing community tools for use in data processing pipelines  
+* Working with the community to provide feedback on metadata standards for raw data in the HCA DCP to provide the required information to enable processing of the raw data  
+* Evaluating and adopting file formats and associated standards for outputs of pipelines, ensuring the data is accessible with standard tools and interoperable with downstream processes  
+
+### Pipeline Benchmarking
+
+* Investigating and identifying data for pipeline benchmarking  
+* Establishing and operating automated practices around scientific testing of pipelines  
+* Establishing standard benchmarking used to evaluate data processing pipelines  
+
+### Pipeline Improvement
+
+* Updating pipelines ran in the HCA DCP for optimization of cost, time, or scientific benchmarks  
+
+### Support
+
+* Performing outreach to users on the usability of data in downstream applications  
+* Performing outreach to developers on pipeline development in the HCA DCP  
+* Working with the metadata team and data consumers to ensure the metadata provided about pipelines and the analysis results meet the needs of downstream users  
+* Creating tools that support the creation of community pipelines - may include software, documentation or support  
+
+## Scientific "guardrails"
+
+* Initial versions of standardized pipelines are reviewed by the AWG.  
+* Changes to standardized pipelines that change the scientific output are reviewed by the AWG, engineering updates that do not change the scientific output are not reviewed by the AWG.  
```

Comment:
Editorial suggestions:

_~Changes~ Improvements to standardized pipelines that change the scientific output must be reviewed by the AWG. Engineering updates that do not change the scientific output do not require review by the AWG._

Reply:
Updated to "Updates to standardized pipelines that change the scientific output must be reviewed by the AWG. Engineering     updates that do not change the scientific output do not require review by the AWG."